tests/cli/lib/lib.sh

```diff
@@ -108,7 +108,6 @@ composer_start() {
         # socket stop/start seems to be necessary for a proper service restart
         # after a previous direct manual run for it to work properly
         systemctl start $BACKEND.socket
-        systemctl start $BACKEND
```

Comment:
It feels a bit weird that the commit message  says "start osbuild-composer.socket" for this removal?